### PR TITLE
fix: Show fair booth when it's full featured only

### PR DESCRIPTION
**Pull request by anastasiapyzhik**
https://artsyproduct.atlassian.net/browse/FX-2535

**Before**
Screen Shot 2021-06-10 at 6 47 53 PM

**After**
Screen Shot 2021-06-10 at 6 48 36 PM

### CODE REVIEW

File: src/v2/Apps/Show/ShowApp.tsx [hunk 1]
```diff
@@ -104,8 +104,12 @@ export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
               <ShowArtworksEmptyState show={show} />
             </>
           )}
-          <Separator as="hr" my={6} />
-          <ShowContextCard show={show} />
+          {show.fair?.hasFullFeature !== false && (
```
Comment (dblandin) [inline src/v2/Apps/Show/ShowApp.tsx:107]: **minor / non-blocking**: I think a truthy conditional would work just as well here:

```suggestion
          {show.fair?.hasFullFeature && (
```

**Review by dblandin — APPROVED** [review]
Looks great! One minor suggestion which you can take or leave. Assigning back to you so you can merge when ready. 💯

File: src/v2/Apps/Show/ShowApp.tsx (hunk 1 above)
Comment (anastasiapyzhik) [reply src/v2/Apps/Show/ShowApp.tsx:107]: That was intended. Fair may be either object with `hasFullFeature` prop or `null`.
For instance here https://www.artsy.net/show/loft-art-gallery-mous-lamrabat-digital-solo-show?sort=partner_show_position fair is `null` (hence `hasFullFeature` is `falsy`) but link is not broken, it just refers to _/loft-art-gallery_, not to _/fair/loft-art-gallery_

Comment (dblandin) [reply src/v2/Apps/Show/ShowApp.tsx:107]: Ahh, that makes sense. Thanks for clarifying!!

**Pull request merged**